StrongInject.Tests.Integration/TestCircularFuncDependencies.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace StrongInject.Tests.Integration
+{
+    public partial class TestCircularFuncDependencies
+    {
+        public record A(int I, Func<string, B> Func);
+
+        public record B(string S, Func<int, A> Func);
+
+        [Register<A>()]
+        [Register<B>()]
```

Comment:
Whoa, so that happened! Cool.
Are the parens required when using generic attributes?

Reply:
The parens are not required, good catch.